tests/tracer/test_sampler.py

```diff
@@ -261,6 +261,62 @@ def test_sampling_rule_init():
     assert rule.name == name_regex
 
 
+def test_sampling_rule_init_via_env():
+    # Testing single sampling rule
+    with override_env(dict(DD_TRACE_SAMPLING_RULES='[{"sample_rate":1.0,"service":"xyz","name":"abc"}]')):
+        sampling_rule = DatadogSampler().rules
```

Comment:
we should probably assert on the number of rules as well, `assert len(sampling_rules) == 1`

Reply:
I'll add those in!